design-documents/testing/functional/versioning-and-backward-compatibility-policy.md

```diff
@@ -28,82 +31,75 @@ An approach of defining what each release should include was taken from [Semanti
+ ⃰ - `<action>` === (actionGroup, acceptPopup, actionGroup, amOnPage, amOnUrl, amOnSubdomain, appendField, assertArrayIsSorted, assertArraySubset, assertElementContainsAttribute, attachFile, cancelPopup, checkOption, clearField, click, clickWithLeftButton, clickWithRightButton, closeAdminNotification, closeTab, comment, conditionalClick, createData, deleteData, updateData, getData, dontSee, dontSeeJsError, dontSeeCheckboxIsChecked, dontSeeCookie, dontSeeCurrentUrlEquals, dontSeeCurrentUrlMatches, dontSeeElement, dontSeeElementInDOM, dontSeeInCurrentUrl, dontSeeInField, dontSeeInFormFields, dontSeeInPageSource, dontSeeInSource, dontSeeInTitle, dontSeeLink, dontSeeOptionIsSelected, doubleClick, dragAndDrop, entity, executeJS, executeInSelenium, fillField, formatMoney, generateDate, grabAttributeFrom, grabCookie, grabFromCurrentUrl, grabMultiple, grabPageSource, grabTextFrom, grabValueFrom, loadSessionSnapshot, loginAsAdmin, magentoCLI, makeScreenshot, maximizeWindow, moveBack, moveForward, moveMouseOver, mSetLocale, mResetLocale, openNewTab, pauseExecution, parseFloat, performOn, pressKey, reloadPage, resetCookie, submitForm, resizeWindow, saveSessionSnapshot, scrollTo, scrollToTopOfPage, searchAndMultiSelectOption, see, seeCheckboxIsChecked, seeCookie, seeCurrentUrlEquals, seeCurrentUrlMatches, seeElement, seeElementInDOM, seeInCurrentUrl, seeInField, seeInFormFields, seeInPageSource, seeInPopup, seeInSource, seeInTitle, seeLink, seeNumberOfElements, seeOptionIsSelected, selectOption, setCookie, submitForm, switchToIFrame, switchToNextTab, switchToPreviousTab, switchToWindow, typeInPopup, uncheckOption, unselectOption, wait, waitForAjaxLoad, waitForElement, waitForElementChange, waitForElementNotVisible, waitForElementVisible, waitForJS, waitForLoadingMaskToDisappear, waitForPageLoad, waitForText, assertArrayHasKey, assertArrayNotHasKey, assertArraySubset, assertContains, assertCount, assertEmpty, assertEquals, assertFalse, assertFileExists, assertFileNotExists, assertGreaterOrEquals, assertGreaterThan, assertGreaterThanOrEqual, assertInstanceOf, assertInternalType, assertIsEmpty, assertLessOrEquals, assertLessThan, assertLessThanOrEqual, assertNotContains, assertNotEmpty, assertNotEquals, assertNotInstanceOf, assertNotNull, assertNotRegExp, assertNotSame, assertNull, assertRegExp, assertSame, assertStringStartsNotWith, assertStringStartsWith, assertTrue, expectException, fail, dontSeeFullUrlEquals, dontSee, dontSeeFullUrlMatches, dontSeeInFullUrl, seeFullUrlEquals, seeFullUrlMatches, seeInFullUrl, grabFromFullUrl)
```

Comment:
Need to think about procedure of maintaining this list up-to-date. Or better give reference to externally defined list.

Reply:
I'll give reference to the list defined in MFTF documentation instead to remove need to keep it in sync.